plugins/reporters/opossum/src/main/kotlin/OpossumReporter.kt

```diff
@@ -355,8 +356,7 @@ class OpossumReporter(
                     val license = licenseFindings
                         .filter { it.location.path == pathFromFinding }
                         .map { it.license }
-                        .distinct()
-                        .reduceRightOrNull { left, right -> left and right }
+                        .andOrNull()
```

Comment:
> As of [1] each `and()` execution started to invoke `equals()`

Are you referring to the removed `distinct()` call or where was this equality check introduced?

Reply:
[https://github.com/oss-review-toolkit/ort/commit/31b9be83c5793954a7d052a0f1e7e3ca4d9384bd#diff-7e886df65dc15bd3950bdd[…]c6041b357ba7d927debb539aR210](https://github.com/oss-review-toolkit/ort/commit/31b9be83c5793954a7d052a0f1e7e3ca4d9384bd#diff-7e886df65dc15bd3950bdde111cc720070939d64c6041b357ba7d927debb539aR210)
[12:09](https://oss-review-toolkit.slack.com/archives/DNNE8BP44/p1739185775515639)
under hier: [https://github.com/oss-review-toolkit/ort/commit/31b9be83c5793954a7d052a0f1e7e3ca4d9384bd#diff-7e886df65dc15bd3950bdd[…]c6041b357ba7d927debb539aR204](https://github.com/oss-review-toolkit/ort/commit/31b9be83c5793954a7d052a0f1e7e3ca4d9384bd#diff-7e886df65dc15bd3950bdde111cc720070939d64c6041b357ba7d927debb539aR204)